cups/dest.c

```diff
@@ -4183,14 +4191,14 @@ cups_get_dests(
       if ((dest = cupsGetDest(name, instance, num_dests, *dests)) == NULL)
       {
        /*
-	* Out of memory!
-	*/
+	      * Out of memory!
+	      */
```

Comment:
@pranjanpr looks like unintended change, would you mind reverting it?

You can always check the current changes in git before commit them by ```git diff``` (if the changes haven't been staged yet) or ```git diff --staged``` (if you already staged the changes).